Corona-Warn-App/build.gradle

```diff
@@ -455,6 +455,7 @@ dependencies {
     // SECURITY
     implementation "androidx.security:security-crypto:1.0.0-rc03"
     implementation 'org.conscrypt:conscrypt-android:2.5.1'
+    implementation 'org.bouncycastle:bcpkix-jdk15on:1.56'
```

Comment:
Why `1.56` specifically? It's from 2016:
https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk15on